.github/workflows/close-no-repro.yml

```diff
@@ -0,0 +1,27 @@
+name: Close old issues with no-reproduction label
+
+on:
+    schedule:
+        # 08:00 every monday
+        - cron: 0 8 * * 1
+
+jobs:
+    find-issues:
+        name: Search and destroy
+
+        runs-on: ubuntu-latest
+        permissions:
+            issues: write
+            pull-requests: write
+        steps:
+            - uses: actions/stale@v5
+              with:
+                  only-issue-labels: ⁉️ no-reproduction
```

Comment:
Kanskje dette også skal være på norsk, siden de andre labelene våre er det? Hva med "Kan ikke reproduseres" eller "Ikke reproduserbart"? Liker emoji-en, da 😁

Reply:
Vi har labels for "breaking", "dependencies", "epic" og "round robin" så det er ingen godt definert mønsterpraksis for språk på labels. Jeg syns smertegrensa for antall ord er 2 så "kan-ikke-gjenskape" høres litt mye ut for meg. "ugjennskapbar" er kanskje i god tråd med fornorskingstrenden på skjermene ute i fellesområdet?